### PR TITLE
add Nebraska vaccine demographic scrapers

### PR DESCRIPTION
**Pull request by smcclure17**
Dashboard Source: https://experience.arcgis.com/experience/ece0db09da4d4ca68252c3967aa1e9dd/page/page_1/

Add scrapers to track `total_vaccine_initiated`, `total_vaccine_completed`, and `total_vaccine_doses_administered` by race, sex, age, and ethnicity.

### Variables Tracked:
- `attributes.TotalFirstDose` + `attributes.TotalSecondDose` + `attributes.TotalSingleDose` : 
     - _description_: sum each category to get all initiating
     - _tracked as_: `total_vaccine_initiated`
- `attributes.TotalCompletedVac`: 
     - _description_: No description is given, but it appears to include completed second dose series _and_ completed single dose
     - _tracked as_: `total_vaccine_completed`
- `attributes.TotalDoses`: 
     - _description_: "The number of vaccinations given at a Nebraska location"
     - _tracked as_: `total_vaccine_doses_administered`

### Other Variables Supplied:
Other variables are supplied and could be tracked, but are not in our current schema. Including:
- `attributes.TotalSingleDose`: 
     - _description_: doses of single-shot series vaccines
- `attributes.TotalSecondDose`: 
     - _description_: total number of second dose vaccinations (_not_ including single-dose completions)

### CODE REVIEW

**Comment by tlyon3** [comment]
@smcclure17, could you resolve merge conflicts and request a review?

File: can_tools/scrapers/official/NE/ne_vaccines.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import pandas as pd
+import us
+import requests
+
+from can_tools.scrapers import variables
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class NebraskaVaccineSex(StateDashboard):
```
Comment (tlyon3) [inline can_tools/scrapers/official/NE/ne_vaccines.py:9]: Unable to request data. Getting some http errors

File: can_tools/scrapers/official/NE/ne_vaccines.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            "age",
+            "race",
+            "ethnicity",
+            "sex",
+            "value",
+        ]
+        return out.loc[:, cols_to_keep]
+
+
+class NebraskaVaccineRace(NebraskaVaccineSex):
```
Comment (tlyon3) [inline can_tools/scrapers/official/NE/ne_vaccines.py:100]: HTTP errors as well

File: can_tools/scrapers/official/NE/ne_vaccines.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        "race": {
+            "american indian or alaska native": "ai_an",
+            "black or african-american": "black",
+            "native hawaiian or other pacific islander": "pacific_islander",
+            "other race": "other",
+        }
+    }
+
+
+class NebraskaVaccineEthnicity(NebraskaVaccineSex):
```
Comment (tlyon3) [inline can_tools/scrapers/official/NE/ne_vaccines.py:113]: HTTP errors as well

**Review by tlyon3 — CHANGES_REQUESTED** [review]

**Review by sglyon — APPROVED** [review]
@tlyon3 these all worked for me, I think the problem might be with access from foreign country?

I'm going to merge

**Comment by tlyon3** [comment]
Ok I have had that problem with some dashboards before. I've been able to access them by connecting to a VPN, but it didn't work this time. Sounds like it's just a problem with that

**Pull request merged**